Error: Cannot find module '@actions/core'

The following error started to appear when using `vlaurin/action-ghcr-prune@main`

```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@actions/core'
Require stack:
- /home/runner/work/_actions/vlaurin/action-ghcr-prune/main/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/runner/work/_actions/vlaurin/action-ghcr-prune/main/index.js:1:14)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:[12](https://github.com/batmaid/ops/actions/runs/3301276607/jobs/5450659847#step:2:13))
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/vlaurin/action-ghcr-prune/main/index.js'
  ]
}
```

Most likely, it was caused by [this commit](https://github.com/vlaurin/action-ghcr-prune/commit/de0e395685b43d2b34f80d5d280bd76d7eefa7c3).

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -31,6 +31,9 @@ jobs:
       - name: Build
         run: npm run build
 
+      - name: Clear node modules
+        run: rm -rf node_modules/
+
       - name: Should prune untagged versions
         uses: ./ # Itself
         id: test_prune_untagged

--- a/action.yml
+++ b/action.yml
@@ -52,4 +52,4 @@ output:
     description: 'Count of container versions that were pruned'
 runs:
   using: 'node16'
-  main: 'index.js'
+  main: 'dist/index.js'

